pkgs/shells/fish/oh-my-fish/default.nix

```diff
@@ -2,26 +2,24 @@
 , stdenv
 , fetchFromGitHub
 , fish
-, bash
 , runtimeShell
 , writeShellScript
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
```

Comment:
If the function version of mkDerivation here used for some specific reason? I don't see it used anywhere else in nixpkgs, so curious about the change.

Reply:
Personal taste. 
I believe `rec` should be deprecated (https://nix.dev/anti-patterns/language#rec-expression), and now that this style of recursive attributes is supported, I am using it instead.
It is more clear than the `let ... in` that the article linked above suggests.